pages/blog/february-2021-at-asyncapi.md

```diff
@@ -0,0 +1,188 @@
+
+Please take your time to:
+- Read [Finding a Good Open Governance Model for AsyncAPI](/blog/governance-motivation) to understand our motivation
+- Check out [this](https://github.com/asyncapi/.github/pull/37) pull request with the charter
+
+## Code generators activities
+
+We had some significant traffic in the area of code generation templates in February.
+
+### PHP
+
+[Emiliano Zublena](https://twitter.com/emilianozublena) joined the AsyncAPI community big time by starting with donating [a new template for PHP](https://github.com/asyncapi/asyncapi-php-template). It is not yet released under **@asyncapi** scope on npm, but you can already play with it by using the AsyncAPI Generator with direct GitHub link like:
+
+```bash
+ag https://bit.ly/asyncapi https://github.com/asyncapi/asyncapi-php-template -o output`
+```
+
+### Go
+
+We merged the initial pull request to the [Go template](https://github.com/asyncapi/go-template). The initial contributor was not able to continue working on the template, but the foundation is there. [Emiliano Zublena](https://twitter.com/emilianozublena) and [Takumi Sueda](https://github.com/puhitaku) will try to help to drive forward template development. This template is not yet released as we need to get some feedback from the community first. Give it a try with:
+
+```bash
+ag https://bit.ly/asyncapi https://github.com/asyncapi/go-template -o output
+```
+Let us know what you think in the GitHub issues section. Thank you [Jacob Poston](https://github.com/jposton96a) for your initial hard work on the template!
+
+### TypeScript and NATS
+
+[Jonas Lagoni](https://github.com/jonaslagoni/) regularly extends and polishes (the TypeScript template for NATS](https://github.com/asyncapi/ts-nats-template). It reached 0.3 release and is the first template that is already using a new AsyncAPI SDK for data types generation. Please go and check it out.
+
+## AsyncAPI CLI to rule them all
+
+We always wanted to have a single CLI to do all the things related to AsyncAPI. In other words, one CLI to validate, generate, edit, create and others. So far, we had CLI for the AsyncAPI generator, and recently one of the community members, [Jorge Aguiar Martín](https://twitter.com/jotamusik), referred us to his CLI for AsyncAPI validation.
+
+We all agreed that instead of working separately on different CLIs, it is better to work together on one CLI that everyone will love. The project kicked off! Repository created, and we already discuss details on how the CLI interface should look like. [Join](https://github.com/asyncapi/cli/issues/1) the discussion.
```

Comment:
```suggestion
We all agreed that instead of working separately on different CLIs, it is better to work together on one CLI that everyone will love. The project kicked off! A repository has been created and we already discuss details on how the CLI interface should look like. [Join](https://github.com/asyncapi/cli/issues/1) the discussion.
```